index.html

```diff
@@ -65,12 +102,39 @@ <h2>Example Modal 2</h2>
         </noscript>
       </div>
     </div>
+    <style>
+      .unl .dcf-tabs-list-item {
+        display: inline-block;
+      }
+      .unl .dcf-tab {
+        font-family: 'Gotham SSm A', 'Gotham SSm B';
+        font-size: .84em;
+        font-weight: bold;
+        padding: 1em 1.33em .75em;
+        text-decoration: none;
+      }
+      .unl .dcf-tab:hover {
+        text-decoration: underline;
+      }
+      .unl .dcf-tab[aria-selected] {
+        background-color: #eee;
+        color: #424240;
+      }
+      .unl .dcf-tabs-panel {
+        background-color: #eee;
+        padding: 1.78em 1.78vw;
+      }
+    </style>
```

Comment:
Because these styles are specific to the University of Nebraska–Lincoln theme, they can be omitted. I'll create some default styles for the starter theme in a separate pull request. Thanks.